<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>771</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ClientPage">
     <widget class="QTableView" name="ClienttableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>751</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    border: 1px solid #ccc;
    gridline-color: #ccc;
    background-color: #f5f5f5;
}

QTableView::item {
    padding: 5px;
    border: 1px solid #F0ECE5;
}

QHeaderView::section {
    background-color: #F0ECE5;
    color: #161A30;
    padding: 5px;
    border: 1px solid #ddd;
}

QTableView::item:selected {
    background-color: #F0ECE5;
}</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>460</y>
        <width>421</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QPushButton" name="UpdateButton">
         <property name="styleSheet">
          <string notr="true">/* Estilo para los botones (QPushButton) */
QPushButton {
    font-size: 16px;
    background-color: #F0ECE5; /* Fondo marrón */
    color: black; /* Texto color negro */
    border: none;
	padding : 3px;
    border-radius: 0;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    margin: 4px 2px;
    cursor: pointer;
    transition: background-color 0.3s, color 0.3s; /* Transición suave */
}

/* Cambio de color al pasar el ratón sobre el botón */
QPushButton:hover {
    background-color: #31304D; 
    color: white; 
	border : 1px solid #F0ECE5;
    border-radius: 5px;
}</string>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="NewButton">
         <property name="styleSheet">
          <string notr="true">/* Estilo para los botones (QPushButton) */
QPushButton {
    font-size: 16px;
    background-color: #F0ECE5; /* Fondo marrón */
    color: black; /* Texto color negro */
    border: none;
    border-radius: 0;
	padding : 3px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    margin: 4px 2px;
    cursor: pointer;
    transition: background-color 0.3s, color 0.3s; /* Transición suave */
}

/* Cambio de color al pasar el ratón sobre el botón */
QPushButton:hover {
    background-color: #31304D; 
    color: white; 
	border : 1px solid #F0ECE5;
    border-radius: 5px;
}
</string>
         </property>
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="DeleteButton">
         <property name="styleSheet">
          <string notr="true">/* Estilo para los botones (QPushButton) */
QPushButton {
    font-size: 16px;
    background-color: #F0ECE5; /* Fondo marrón */
    color: black; /* Texto color negro */
    border: none;
    border-radius: 0;
padding : 3px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    margin: 4px 2px;
    cursor: pointer;
    transition: background-color 0.3s, color 0.3s; /* Transición suave */
}

/* Cambio de color al pasar el ratón sobre el botón */
QPushButton:hover {
    background-color: #31304D; 
    color: white; 
	border : 1px solid #F0ECE5;
    border-radius: 5px;
}
</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="AdminPage">
     <widget class="QTableView" name="AdminsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>731</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    border: 1px solid #ccc;
    gridline-color: #ccc;
    background-color: #f5f5f5;
}

QTableView::item {
    padding: 5px;
    border: 1px solid #F0ECE5;
}

QHeaderView::section {
    background-color: #F0ECE5;
    color: #161A30;
    padding: 5px;
    border: 1px solid #ddd;
}

QTableView::item:selected {
    background-color: #F0ECE5;
}</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>460</y>
        <width>359</width>
        <height>37</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="NewAdminButton">
         <property name="styleSheet">
          <string notr="true">/* Estilo para los botones (QPushButton) */
QPushButton {
    font-size: 16px;
    background-color: #F0ECE5; /* Fondo marrón */
    color: black; /* Texto color negro */
    border: none;
    border-radius: 0;
padding : 3px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    margin: 4px 2px;
    cursor: pointer;
    transition: background-color 0.3s, color 0.3s; /* Transición suave */
}

/* Cambio de color al pasar el ratón sobre el botón */
QPushButton:hover {
    background-color: #31304D; 
    color: white; 
	border : 1px solid #F0ECE5;
    border-radius: 5px;
}</string>
         </property>
         <property name="text">
          <string>New Admin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="DeleteAdminButton">
         <property name="styleSheet">
          <string notr="true">/* Estilo para los botones (QPushButton) */
QPushButton {
    font-size: 16px;
    background-color: #F0ECE5; /* Fondo marrón */
    color: black; /* Texto color negro */
    border: none;
    border-radius: 0;
padding : 3px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    margin: 4px 2px;
    cursor: pointer;
    transition: background-color 0.3s, color 0.3s; /* Transición suave */
}

/* Cambio de color al pasar el ratón sobre el botón */
QPushButton:hover {
    background-color: #31304D; 
    color: white; 
	border : 1px solid #F0ECE5;
    border-radius: 5px;
}</string>
         </property>
         <property name="text">
          <string>Delete Admin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="UpdateAdminsButton">
         <property name="styleSheet">
          <string notr="true">/* Estilo para los botones (QPushButton) */
QPushButton {
    font-size: 16px;
    background-color: #F0ECE5; /* Fondo marrón */
    color: black; /* Texto color negro */
    border: none;
    border-radius: 0;
padding : 3px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    margin: 4px 2px;
    cursor: pointer;
    transition: background-color 0.3s, color 0.3s; /* Transición suave */
}

/* Cambio de color al pasar el ratón sobre el botón */
QPushButton:hover {
    background-color: #31304D; 
    color: white; 
	border : 1px solid #F0ECE5;
    border-radius: 5px;
}</string>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Commande">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>20</y>
        <width>601</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item row="4" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="CommandeButton">
       <property name="styleSheet">
        <string notr="true">/* Estilo para los botones (QPushButton) */
QPushButton {
    font-size: 16px;
    background-color: #F0ECE5; /* Fondo marrón */
    color: black; /* Texto color negro */
    border: none;
    border-radius: 0;
	padding : 3px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    margin: 4px 2px;
    cursor: pointer;
    transition: background-color 0.3s, color 0.3s; /* Transición suave */
}

/* Cambio de color al pasar el ratón sobre el botón */
QPushButton:hover {
    background-color: #31304D; 
    color: white; 
	border : 1px solid #F0ECE5;
    border-radius: 5px;
}</string>
       </property>
       <property name="text">
        <string>Commande</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="AdminButton">
       <property name="styleSheet">
        <string notr="true">/* Estilo para los botones (QPushButton) */
QPushButton {
    font-size: 16px;
    background-color: #F0ECE5; /* Fondo marrón */
    color: black; /* Texto color negro */
    border: none;
    border-radius: 0;
	padding : 3px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    margin: 4px 2px;
    cursor: pointer;
    transition: background-color 0.3s, color 0.3s; /* Transición suave */
}

/* Cambio de color al pasar el ratón sobre el botón */
QPushButton:hover {
    background-color: #31304D; 
    color: white; 
	border : 1px solid #F0ECE5;
    border-radius: 5px;
}</string>
       </property>
       <property name="text">
        <string>Admins</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="ClientButton">
       <property name="styleSheet">
        <string notr="true">/* Estilo para los botones (QPushButton) */
QPushButton {
    font-size: 16px;
    background-color: #F0ECE5; /* Fondo marrón */
    color: black; /* Texto color negro */
    border: none;
    border-radius: 0;
	padding : 3px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    margin: 4px 2px;
    cursor: pointer;
    transition: background-color 0.3s, color 0.3s; /* Transición suave */
}

/* Cambio de color al pasar el ratón sobre el botón */
QPushButton:hover {
    background-color: #31304D; 
    color: white; 
	border : 1px solid #F0ECE5;
    border-radius: 5px;
}</string>
       </property>
       <property name="text">
        <string>Client</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="CategorieButton">
       <property name="styleSheet">
        <string notr="true">/* Estilo para los botones (QPushButton) */
QPushButton {
    font-size: 16px;
    background-color: #F0ECE5; /* Fondo marrón */
    color: black; /* Texto color negro */
    border: none;
    border-radius: 0;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    margin: 4px 2px;
    cursor: pointer;
    transition: background-color 0.3s, color 0.3s; /* Transición suave */
}

/* Cambio de color al pasar el ratón sobre el botón */
QPushButton:hover {
    background-color: #31304D; 
    color: white; 
	border : 1px solid #F0ECE5;
    border-radius: 5px;
}</string>
       </property>
       <property name="text">
        <string>Categorie</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
